src/d3a/setup/myco_setup/external_myco.py

```diff
@@ -16,40 +16,40 @@ def get_setup(config):
             Area(
                 "House 1",
                 [
-                    Area("H1 General Load1", strategy=LoadHoursStrategy(avg_power_W=200,
-                                                                        hrs_per_day=6,
-                                                                        hrs_of_day=list(
-                                                                            range(12, 18)),
-                                                                        final_buying_rate=35)),
-                    Area("H1 General Load2", strategy=LoadHoursStrategy(
-                        avg_power_W=150,
-                        hrs_per_day=24,
-                        hrs_of_day=list(range(0, 24)),
-                        final_buying_rate=40)),
-                    Area("H1 Storage1", strategy=StorageStrategy(initial_soc=100,
-                                                                 battery_capacity_kWh=20)),
-                    Area("H1 Storage2", strategy=StorageStrategy(initial_soc=100,
-                                                                 battery_capacity_kWh=20)),
-                    Area("H1 PV1", strategy=PVStrategy(panel_count=4)),
-                    Area("H1 PV2", strategy=PVStrategy(panel_count=4)),
+                    Area("H1 General Load1",
+                         strategy=LoadHoursExternalStrategy(avg_power_W=200,
```

Comment:
Can you please explain why you used the external strategies here? AFAIU, this is not needed to run the external myco matcher right?

Reply:
True, this was just in order to also connect the d3a-api-client

Reply:
Ok. We can not rely on manual testing, so I really vote for https://github.com/gridsingularity/d3a/pull/1197#discussion_r685760708